Mathlib/Topology/Separation/Basic.lean

```diff
@@ -227,6 +227,9 @@ protected theorem Topology.IsEmbedding.t0Space [TopologicalSpace Y] [T0Space Y]
 @[deprecated (since := "2024-10-26")]
 alias Embedding.t0Space := IsEmbedding.t0Space
 
+protected theorem Homeomorph.t0Space [TopologicalSpace Y] [T0Space X] (h : X ≃ₜ Y) : T0Space Y :=
```

Comment:
I presume there's no variable `[TopologicalSpace Y]` around that you can use? (Similarly for the other lemmas.)

Reply:
There isn't, and I didn't want to disturb the file further